vendor/k8s.io/kubernetes/plugin/cmd/kube-scheduler/app/server_patch.go

```diff
@@ -0,0 +1,7 @@
+package app
+
+
+func (o *Options) ReallyApplyDefaults() (err error) {
```

Comment:
I'm not sure it's worth the carry, if we use it in two places: one is actual code, and the second is test. Why not just creating a helper in origin?

Reply:
> I'm not sure it's worth the carry, if we use it in two places: one is actual code, and the second is test. Why not just creating a helper in origin?

It's touching a private value in a struct, for a process we're splitting out of origin. This is the simplest change we can do until we stop using it altogether.